CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/docs/import-from-delimited-file.html

```diff
@@ -40,6 +40,7 @@ <h2>High level workflow steps</h2>
         <ul>
             <li>See all supported attributes using Options -&gt; Show all schema attributes</li>
             <li>Hover over the attribute name for a tooltip.</li>
+            <li>Start typing to search as &lt;column_name&gt; =="&lt;search text&gt;" E.g.: first_name=="Nick".</li>
```

Comment:
I'd maybe change this to "Filter in the Configuration Pane by adding searches of the form &lt;column_name&gt; =="&lt;search text&gt;""

Reply:
Remove the e.g."?